vyper/typing.py

```diff
@@ -1,18 +1,21 @@
-from typing import Dict, Optional, Sequence, Tuple, Union
+from tokenize import TokenInfo
+from typing import Optional, Sequence, Union
 
 # Parser
-ModificationOffsets = Dict[Tuple[int, int], str]
-ParserPosition = Tuple[int, int]
+ForLoopAnnotations = dict[tuple[int, int], list[TokenInfo]]
```

Comment:
i think now that we have `PreParseResult`, we don't really need any of these type aliases